python/eups/db/ChainFile.py

```diff
@@ -175,6 +180,10 @@ def write(self, file=None):
 
         fd.close()
 
+    REGEX_KEYVAL = re.compile(r"^(\w+)\s*=\s*(.*)", flags = re.IGNORECASE)
+    REGEX_GROUPEND = re.compile(r"^(End|Group)\s*:")
+    REGEX_NORMALIZE = re.compile(r"^#.*$")
```

Comment:
This is not used, and it doesn't do what the previous regex did, anyway (strip out comments at ends of lines).